### ISSUE TITLE
Element timing report table appears to have shifted columns

### ISSUE THREAD

**Issue by dooglewoogle** [sitespeedio/sitespeed.io#3926]
### Have you read the documentation?

- [X] Yes, I've read the [how to make a reproducable bug guide](https://www.sitespeed.io/documentation/sitespeed.io/bug-report/)
- [X] Yes, I've read the [how to debug my script guide](https://www.sitespeed.io/documentation/sitespeed.io/scripting/#debug)

### URL

localhost

### What are you trying to accomplish

Hello, I'm working on integrating sitespeed.io into some of our monitoring. After standing up a dev instance of the application, and adding the "elementtiming=...." attribute to some of the sections I wanted to test, the results table created is:
<img width="1253" alt="Screenshot 2023-08-02 at 2 37 59 PM" src="https://github.com/sitespeedio/browsertime/assets/46539436/d2e5de30-71f2-473b-a0f7-6beee2ce64fe">
This doesn't look right to me. My speculation is that the "URL" and "Image" columns have had no data populated, and what data does exist has been left-shifted. For example: 'H1' should be under "Tag", and '6.415s' should be under "Render Time", and so on. The data itself looks fine, just in the wrong place. 

Does this assessment seem valid, or is this what you would expect to see? Following on, is there any further things I can try, either to get more information, or to perhaps resolve it? Unfortunately I haven't been able to make this publicly reproducible. 

For context, this particular project is a React app, was tested in a standalone dev environment. We are investigating using element timing as this part of the platform will be gated behind other logins and containers, as well as loaded after initial page load. 

Thanks. 

### What browser did you use?

Chrome

### How to reproduce

```shell
sitespeed.io http://testlocalserver -b chrome --chrome.binaryPath /usr/bin/chromium-browser -n 1

Sitespeed: 29.1.2
Chromium: 113.0.5672.63 
OS: Fedora 36 ARM VM
```

### Log output

```shell
[2023-08-02 15:57:51] INFO: Versions OS: linux 6.2.15-100.fc36.aarch64 nodejs: v16.18.1 sitespeed.io: 29.1.2 browsertime: 17.14.1 coach: 7.2.1
[2023-08-02 15:57:52] INFO: Running tests using Chrome - 1 iteration(s)
[2023-08-02 15:57:55] INFO: Testing url http://testlocalserver iteration 1
[2023-08-02 15:58:02] INFO: Take after page complete check screenshot
[2023-08-02 15:58:03] INFO: Take cumulative layout shift screenshot
[2023-08-02 15:58:03] INFO: Take largest contentful paint screenshot
[2023-08-02 15:58:04] INFO: Pausing for breakpoint End-of-iteration, set browsertime.pause = false; to continue
[2023-08-02 15:58:25] INFO: Exit breakpoint and continue.
[2023-08-02 15:58:28] INFO: http://testlocalserver 53 requests, TTFB: 61ms, firstPaint: 91ms, FCP: 91ms, DOMContentLoaded: 191ms, LCP: 1.69s, CLS: 0.5254, TBT: 741ms, CPUBenchmark: 45ms, Load: 191ms
[2023-08-02 15:58:29] INFO: The server responded with a 404 status code for http://testlocalserver/favicon.ico
[2023-08-02 15:58:30] INFO: HTML stored in /Users/dooglewoogle/sitespeed-result/localtest/2023-08-02-15-57-51
```

**Comment by soulgalore** [comment]
Hi @dooglewoogle thanks for creating the issue. You are right. There's a bug when an element is not an image. I fixed it in #3927 and will roll it out this weekend.

**Comment by soulgalore** [comment]
This should be fixed in 29.1.3.

**Comment by dooglewoogle** [comment]
Just pulled 29.1.3 and tested. Fixed and working. Thank you so much for doing this so quickly!
